app.js

```diff
@@ -12,9 +14,10 @@ app.get("/api", (req, res) => {
 });
 
 app.get("/api/topics", getAllTopics); //create router
+app.get("/api/articles", getAllArticles);
 
 const topicName = "topicz";
-app.use(`/api/${topicName}`, errorHandler404);
+app.use(`/api/:topicName`, errorHandler404);
```

Comment:
this error handling is not correct as it will send you user to a 404 if there are going to /api/*